Mathlib/NumberTheory/Ostrowski.lean

```diff
@@ -0,0 +1,64 @@
+/-
+Copyright (c) 2024 María Inés de Frutos-Fernández. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: David Kurniadi Angdinata, Fabrizio Barroero, Laura Capuano, Nirvana Coppola,
+María Inés de Frutos Fernández, Sam van Gool, Silvain Rideau-Kikuchi, Amos Turchet,
+Francesco Veneziano
+-/
+
+import Mathlib.Analysis.Normed.Field.Basic
+import Mathlib.Analysis.SpecialFunctions.Pow.Real
+import Mathlib.Analysis.Normed.Ring.Seminorm
+
+/-!
+# Ostrowski’s Theorem
+
+The goal of this file is to prove Ostrowski’s Theorem which gives a list of all the nontrivial
+absolute values on a number field up to equivalence.
+
+## References
+* https://kconrad.math.uconn.edu/blurbs/gradnumthy/ostrowskiQ.pdf
+* https://kconrad.math.uconn.edu/blurbs/gradnumthy/ostrowskinumbfield.pdf
+
+## Tags
+ring_norm, ostrowski
+-/
+
+noncomputable section
+namespace Rat.MulRingNorm
+open Int
+
+variable {f g : MulRingNorm ℚ}
+
+/-- Values of a multiplicative norm of the rationals are determined by the values on the
+integers. -/
+lemma NormRat_eq_on_Int_iff_eq_on_Nat : (∀ n : ℕ , f n = g n) ↔ (∀ n : ℤ , f n = g n) := by
```

Comment:
I am not sure that this names adheres to the convention (see https://leanprover-community.github.io/contribute/naming.html). Why is `NormRat` upper case? I cannot find any `structure` in `mathlib` with this name, am I wrong?

Reply:
I implemented Maria's suggestion to remove "NormRat" for the name at all!